pkg/operator/status.go

```diff
@@ -28,15 +28,25 @@ func (optr *Operator) syncAvailableStatus() error {
 	}
 
 	optrVersion, _ := optr.vStore.Get("operator")
+
+	var progressing bool
+	if !optr.vStore.Equal(co.Status.Versions) {
+		progressing = true
```

Comment:
One thing I'm wondering here...we want `progressing=true` when `optr.inClusterBringup` too right?
IOW this code is handling the upgrade case but we also want to be progressing during initial install.

Reply:
good call, I'm gonna work that out

Reply:
fixed here https://github.com/openshift/machine-config-operator/pull/406/files#diff-bee1f8f36240cb95db10244fdf335146R68 - we now set progressing = true on bring up as well

Reply:
One thing I've noticed related to this though is that we seem to stay in `inClusterBringup` for multiple times.  Though maybe that's only failing PRs?  Glancing at [the logs in a recent merged PR https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/419/pull-ci-openshift-machine-config-operator-master-e2e-aws/1756/artifacts/e2e-aws/pods/] things look fine.

Reply:
mmm what do you mean "stay in `inClusterBringup` for multiple times"?

Reply:
If the sync fails; you'll see that log multiple times.  But I think that's the right thing?

Reply:
oh yeah, it's correct we keep re-syncing and logging that